### ISSUE TITLE
5e - Artificer

### ISSUE THREAD

**Issue by chainria** [BahamutDragon/pcgen#1195]
Hello and thank you for the great work on these sources. Many a session wouldn't have been possible without them.

While trying to create an Artificer, we stumbled upon several issues:

- The only way right now to create an Artificer is by using UA - the version from Tasha's is completely missing. Unfortunately, there are subtle differences, but that's not that big a problem.
- Multiclass Allowed is not set for the UA Artificer. Which basically means, there's not Multiclassing in or out of Artificer. That's quite a biggie though and should be easy to fix.

Could we maybe get one or both of these issues fixed? That would be very great and also very appreciated.

**Comment by BahamutDragon** [comment]
I can fix the multi-class concern. Coding up a subtle difference class might take longer.

**Comment by BahamutDragon** [comment]
3 hours worth of work there. 2 of the 4 specialties are good to go. You have spells. Multiclass support should now work for the TCE version.

**Comment by chainria** [comment]
Thank you so very much! That's great news! I'll relay it to my DM so we can try it out. Yes, the multiclass concern was the biggest issue for us. That's some solid work there!

**Comment by BahamutDragon** [comment]
Okay, that is it for me. Most of that should work, but no testing yet. If you have any issues, just open a new ticket.

**Comment by chainria** [comment]
Thank you very much! We couldn't find anything wrong today. Will keep testing for a while.
Absolutely great, thanks again!

**Comment by chainria** [comment]
We actually found something. At 3rd Level, there should be a specialist selection. But nothing comes up and we can't seem to find anything under feats etc. either. I took a look at the sources and it *should* technically be there, but nothing seems to pop. I tried an empty test character as well, same thing. Under 6.08 here.

If I understood correctly, the selection should be implemented, but not the actual specializations themselves. Is that right? Either way, we can live with what we have for now, I just wanted to document the state as promised :)
